<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventana</class>
 <widget class="QWidget" name="Ventana">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>411</width>
     <height>641</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lOrigen">
      <property name="text">
       <string>Origen :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leOrigenLat"/>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pbBuscar">
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lDestino">
      <property name="text">
       <string>Destino:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="leDestinoLat">
      <property name="placeholderText">
       <string comment="Latitud" extracomment="Latitud"/>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="lTiempo">
      <property name="text">
       <string>Tiempo :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="leTiempo"/>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="lDistancia">
      <property name="text">
       <string>Distancia:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLineEdit" name="leDistancia"/>
    </item>
    <item row="6" column="0" colspan="4">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leOrigenLng"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="leDestinolng"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
